frontend/src/components/landing/carousel/ContentCards.tsx

```diff
@@ -0,0 +1,36 @@
+import styles from "./ContentCards.module.scss";
+
+export type Props = {
+  heading: string;
+  lead: string;
+  cards: Array<{
+    image: string;
+    heading: string;
+    body: string;
+    alt: string;
+  }>;
+};
+
+export const CarouselContentCards = (props: Props) => {
+  return (
+    <div className={`${styles.wrapper}`}>
+      <div className={styles.text}>
+        <div className={styles.heading}>{props.heading}</div>
+        <div className={styles.lead}>{props.lead}</div>
+      </div>
+      <div className={styles.hero}>
+        {props.cards.map((card, index) => {
+          return (
+            <div className={styles.card} key={index}>
+              <div className={styles["card-image-container"]}>
+                <img src={card.image} alt={card.alt} />
```

Comment:
As [mentioned in the l10n repository](https://github.com/mozilla-l10n/fx-private-relay-l10n/pull/78#discussion_r862741043), these images are purely decorative, so adding a non-empty alt text can even be detrimental to a11y (though :clap: for thinking about it in the first place - it gets overlooked far too often!):

```suggestion
                <img src={card.image} alt="" />
```

(And then the `alt` in the `Props` can be removed as well.)